packages/firestore/exp/src/api/database.ts

```diff
@@ -250,15 +196,12 @@ export function getFirestore(app: FirebaseApp): FirebaseFirestore {
  * @return A promise that represents successfully enabling persistent storage.
  */
 export function enableIndexedDbPersistence(
-  firestore: FirebaseFirestore,
+  firestore: FirestoreCompat,
   persistenceSettings?: PersistenceSettings
 ): Promise<void> {
   verifyNotInitialized(firestore);
 
-  // `_getSettings()` freezes the client settings and prevents further changes
-  // to the components (as `verifyNotInitialized()` would fail). Components can
-  // then be accessed via `getOfflineComponentProvider()` and
-  // `getOnlineComponentProvider()`
+  const firestoreClient = firestore._ensureClientConfigured();
```

Comment:
There aren't that many different kinds of clients in this code base, so you can probably abbreviate this to just `client` if you wanted to. Here and throughout if you choose to.

Reply:
Mass-renamed `firestoreClient` to `client`.